<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicConnectTransferWidget</class>
 <widget class="QWidget" name="MusicConnectTransferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicConnectTransferWidget</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>615</width>
     <height>369</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>传文件到移动设备</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>615</width>
     <height>344</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(255,255,255,150);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QWidget" name="playListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>0</y>
      <width>361</width>
      <height>344</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,255,255,50)</string>
    </property>
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>5</y>
       <width>165</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/searchline);color:#6495ED;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="allSelectedcheckBox">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>10</y>
       <width>95</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchLineLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>5</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/share/searchlineleft);</string>
     </property>
    </widget>
    <widget class="MusicConnectTransferTableWidget" name="playListTableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>304</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="functionBottomLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>361</width>
       <height>304</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(255,255,255)</string>
     </property>
    </widget>
    <zorder>searchLineEdit</zorder>
    <zorder>allSelectedcheckBox</zorder>
    <zorder>searchLineLabel</zorder>
    <zorder>functionBottomLabel</zorder>
    <zorder>playListTableWidget</zorder>
   </widget>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>72</width>
      <height>72</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../TTKQrc/MusicPlayer.qrc">:/toolSets/usb</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>125</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButton">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>transfer</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicConnectTransferTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicconnecttransfertablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../TTKQrc/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
